src/Messaging/MessageBuffer.php

```diff
@@ -178,6 +180,10 @@ public function onData($data) {
                 $closeFrame = $this->newCloseFrame(Frame::CLOSE_PROTOCOL, 'Invalid frame length');
             }
 
+            if (!$closeFrame && PHP_INT_SIZE == 4 && $payloadLenOver2GB) {
```

Comment:
> PHP Notice:  Undefined variable: payloadLenOver2GB in D:\project\vendor\ratchet\rfc6455\src\Messaging\MessageBuffer.php on line 183

```suggestion
            if (!$closeFrame && PHP_INT_SIZE == 4 && ! empty($payloadLenOver2GB)) {
```

@cboden @Domochip

Reply:
created https://github.com/ratchetphp/RFC6455/pull/66 to fix this issue